packages/typescript/tslint.json

```diff
@@ -17,6 +17,8 @@
     "no-inferrable-types": false,
     "ban-ts-ignore": false,
     "increment-decrement": false,
+    "promise-function-async": false,
+    "ban-types": [true, ["async", "Use Promises instead, it add a lot to bundle size."]],
```

Comment:
Noice

Reply:
Ah, `ban-types` doesn't work :/
`promise-function-async` makes sure we can return `Promise` without the function being async which adds this massive TS helper.